app/api/ddi-index-api/persons.js

```diff
@@ -18,15 +18,16 @@ const options = {
  */
 /**
  * @param {{ firstName: string; lastName: string; dateOfBirth?: string; }} filter
- * @returns {Promise<Person[]>}
+ * @returns {Promise<import('./person.js').Person[]>}
  */
 const getPersons = async (filter) => {
-  const validation = personsFilter.validate(filter, { abortEarly: false })
+  const validation = personsFilter.validate(filter, { abortEarly: false, dateFormat: 'utc', stripUnknown: true })
```

Comment:
In other source files, we've tended to use
const { value, error } = xxx.validate()
to extract value and error. Very minor but it would keep it consistent with all the other source